turbonomic/operands/opensearch-operand-orm.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: devops.turbonomic.io/v1alpha1
+kind: OperatorResourceMapping
+metadata:
+  name: opensearch-orm
+  namespace: $namespace
```

```suggestion
  namespace: {{ placeholder_namespace }}
```

Please change the placeholder value